torch/__init__.py

```diff
@@ -2382,8 +2382,9 @@ def compile(
           There are other circumstances where CUDA graphs are not applicable; use TORCH_LOG=perf_hints
           to debug.
 
-        - "max-autotune" is a mode that leverages Triton based matrix multiplications and convolutions
-          It enables CUDA graphs by default.
+        - "max-autotune" is a mode that leverages CPP template-based matrix multiplications on CPU
+          and Triton based matrix multiplications and convolutions on GPU.
```

Comment:
```suggestion
        - "max-autotune" is a mode that leverages Triton or  template based matrix multiplications 
          on supported devices and Triton based convolutions on GPU.
```

Reply:
Thanks for the suggestion and fix.
Seems there's an issue of the change committed directly when using ghstack. Let me re-submit the change you proposed via ghstack.
![image](https://github.com/user-attachments/assets/a6705fb4-38a5-4255-9131-059850bc1b3c)